internal/controller/ols_app_server_assets.go

```diff
@@ -198,12 +198,12 @@ func (r *OLSConfigReconciler) generateOLSDeployment(cr *olsv1alpha1.OLSConfig) (
 							},
 							Resources: corev1.ResourceRequirements{
 								Requests: corev1.ResourceList{
-									corev1.ResourceCPU:    resource.MustParse("100m"),
```

Comment:
No need to update CPU , just the memory @vishnuchalla

Reply:
Thought `100m` was very less, so bumped that up too. But I can revert it back if we still wanted to keep it low.

Reply:
kubectl top pods was showing this while requests coming in . I would not bring it up unless we see a need for it.
```
NAME                                                                                         CPU(cores)   MEMORY(bytes)   
lightspeed-app-server-69c795d4b6-zrkdg                                1m           1370Mi          
lightspeed-operator-controller-manager-7b9b8696ff-7bbnv   0m           43Mi 
```